src/main/python/ccrf/prepare_training_data.py

```diff
@@ -51,23 +69,19 @@ def build_docid_idx_and_label(rank_file_folder):
     logging.info(f'{cur_idx} files found in total.')
     return docid_idx_dict, topic_docid_label
 
-def build_vocab_tfidf_dict(tfidf_raw_folder, docid_idx_dict, isalpha):
-    """
 
-    """
+def build_vocab_tfidf_dict(docid_idx_dict, isalpha):
```

Comment:
Our fault, please use is_alpha to keep naming consistency.

Reply:
k